java/client/src/main/java/glide/api/commands/servermodules/FT.java

```diff
@@ -474,6 +475,37 @@ public static CompletableFuture<Map<GlideString, Object>[]> aggregate(
                 .thenApply(res -> castArray(res, Map.class));
     }
 
+    /**
+     * Runs a search or aggregation query and collects performance profiling information.
+     *
+     * @param client The client to execute the command.
+     * @param indexName The index name.
+     * @param options Querying and profiling parameters - see {@link FTProfileOptions}.
+     * @return A two-element array. The first element contains results of query being profiled, the
+     *     second element stores profiling information.
```

Comment:
We have examples in other commands but not here. We can add here for consistency.